### PR TITLE
Rolling batch integration tests

### PR DESCRIPTION
**Pull request by sindhuvahinis**
## Description ##

Brief description of what this PR is about

- If this change is a backward incompatible change, why must this change be made?
- Interesting edge cases to note here

### CODE REVIEW

File: tests/integration/rb_client.py [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+# sends multiple requests with different data json concurrently
+def test_concurrent_with_mul_reqs(model, test_spec, spec_name):
+    if model not in test_spec:
+        raise ValueError(
+            f"{args.model} is not one of the supporting models {list(test_spec.keys())}"
+        )
+
+    processes = []
+    spec = test_spec[model]
+    for index in range(len(spec.get("input_texts"))):
```
Comment (lanking520) [inline tests/integration/rb_client.py:116]: why multiple processes and concurrent clients used at the same time?

Comment (sindhuvahinis) [reply tests/integration/rb_client.py:116]: Multiple process with one concurrent client. Need to send different requests with different decoding strategy requests. For example, greedy and contrastive can be sent to the SeqBatchScheduler at the same time. But SeqBatchScheduler will call the forward separately for greedy and contrastive.  This is a scheduler only use case.

File: .github/workflows/rolling_batch_integration.yml [hunk 2]
```diff
@@ -0,0 +1,172 @@
...
+
+on:
+  workflow_dispatch:
+    inputs:
+      djl-version:
+        description: 'The released version of DJL'
+        required: false
+        default: ''
+  schedule:
+    - cron: '0 15 * * *'
```
Comment (lanking520) [inline .github/workflows/rolling_batch_integration.yml:11]: let's schedule a different time, 3PM GMT is too busy.

Comment (sindhuvahinis) [reply .github/workflows/rolling_batch_integration.yml]: Done

**Review by lanking520 — APPROVED** [review]

**Pull request merged**